File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2022-04-01/examples-js/Operations_ListOperations.js
const { PolicyInsightsClient } = require("@azure/arm-policyinsights");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Lists available operations.
 *
 * @summary Lists available operations.
 * x-ms-original-file: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2022-04-01/examples/Operations_ListOperations.json
 */
async function listOperations() {
  const subscriptionId =
    process.env["POLICYINSIGHTS_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
  const credential = new DefaultAzureCredential();
  const client = new PolicyInsightsClient(credential, subscriptionId);
  const result = await client.operations.list();
  console.log(result);
}

File: specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2024-10-01/examples-java/Operations_ListOperations.java

/**
 * Samples for Operations List.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2024-10-01/examples/
     * Operations_ListOperations.json
     */
    /**
     * Sample code: List operations.
     * 
     * @param manager Entry point to PolicyInsightsManager.
     */
    public static void listOperations(com.azure.resourcemanager.policyinsights.PolicyInsightsManager manager) {
        manager.operations().listWithResponse(com.azure.core.util.Context.NONE);
    }
}
